source/wp-content/themes/wporg-main-2022/inc/shortcodes.php

```diff
@@ -95,3 +95,24 @@ function( $attrs = array() ) {
 		return $link;
 	}
 );
+
+/**
+ * Shortcode for a formatted date & time when the latest version was published.
+ */
+add_shortcode(
+	'latest_version_date',
+	function( $attrs = array() ) {
+		$format = $attrs['format'] ?? 'Y-m-d\TH:i:s\+00:00';
+
+		$timestamp = defined( 'WPORG_WP_RELEASES_PATH' ) ? filemtime( WPORG_WP_RELEASES_PATH . 'wordpress-' . WP_CORE_LATEST_RELEASE . '.zip' ) : time();
```

Comment:
Is the fallback here for local envs? If so, it might be safer to do something like:

```
if env == production {
    use timestamp of file
} else {
   use time()
}
```

That way it won't fail silently in production if something goes wrong in the future, and output the wrong time.

That's subjective, though, and definitely not a blocker.

Reply:
Personally, I would be more comfortable checking the constant than the environment type, although in the event that the file doesn't exist this would be `false` anyway.

Although, as the fallback here, I wouldn't be against just returning nothing instead of the current time, as if there's no the appropriate data, it doesn't seem proper to show it.. not that it really matters much IMHO

```suggestion
		$timestamp = false;
		if ( defined( 'WPORG_WP_RELEASES_PATH' ) ) {
			$timestamp = filemtime( WPORG_WP_RELEASES_PATH . 'wordpress-' . WP_CORE_LATEST_RELEASE . '.zip' );
		}
```
plus, then replace `return gmdate( $format, $timestamp );` with something like `return $timestamp ? gmdate( $format, $timestamp ) : '';`